templates/test/comments.html.ep

```diff
@@ -0,0 +1,37 @@
+% if (!$job->comments) {
```

Comment:
I think it's an empty resultset, which is something - so the if doesn't work. But if you use it in array context, it's ->all